qiskit_ibm_runtime/sampler.py

```diff
@@ -42,7 +42,7 @@ class Sampler:
     version = 0
 
 
-class SamplerV2(BasePrimitiveV2[SamplerOptions], Sampler, BaseSamplerV2):
+class SamplerV2(BasePrimitiveV2, Sampler, BaseSamplerV2):
```

Comment:
Why was `SamplerOptions` removed? It's meant to set the expected options class.

Reply:
Seems to me I accidently reverted to an older version. Will fix.